src/technologies/b.json

```diff
@@ -2674,6 +2674,40 @@
     "oss": true,
     "website": "https://fonts.bunny.net"
   },
+  "Burst": {
+    "cats": [
+      10
+    ],
+    "description": "Burst Statistics keeps all data on your server, making it fully compliant with privacy laws. Our dashboards offer clear and concise insights, allowing you to make informed decisions without feeling overwhelmed by abundant data. Choose Burst Statistics for seamless and reliable analytics trusted by over 100,000 users.",
+    "icon": "Burst.svg",
+    "js": {
+      "burst.url": ""
+    },
+    "pricing": [
+      "free"
+    ],
```

Comment:
See [here](https://github.com/HTTPArchive/wappalyzer/tree/main?tab=readme-ov-file#optional-properties) for the list of valid values. If everything that matches this detection is free then I think we can just remove `pricing` all together, otherwise if Premium matches this too we should use `freemium`.

Though to be fair HTTP Archive doesn't use these fields and it's more about conforming to the old (public) Wappalyzer convention.

Reply:
I've dropped the pricing part. Currently it's set up to only match free, and pro has it's own entry

Reply:
Example URLs:

https://complianz.io uses premium (burst-pro)
https://www.ikbeneenkameleon.nl/ uses the free version (burst-statistics)